### PR TITLE
Pretty issue templates

### PR DESCRIPTION
**Pull request by cemremengu**
As stated in #1208 🎨 🖌 

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by StarpTech** [comment]
Awesome! 👍

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -9,35 +11,4 @@ them and confirm that they are a bug or incorrect/missing information in the [do
...
-
-* *node version*: 4,5,6,7,8,9
-* *fastify version*: >=0.37.0
-* *os*: Mac, Windows
-* *any other relevant information*:
-
-**Please read this entire template before posting any issue. If you ignore these instructions
-and post an issue here that does not follow the instructions, your issue might be closed,
-locked, and assigned the `missing discussion` label.**
+If you have issues with the [fastify.io](https://www.fastify.io) site, please open an issue [here](https://github.com/fastify/website/issues).
```
[the remark this answers is not in the document]
Comment (cemremengu) [reply .github/ISSUE_TEMPLATE.md]: You mean I should add another section for `website` to here ?

image

[the remark this answers is not in the document]
Comment (StarpTech) [reply .github/ISSUE_TEMPLATE.md]: No, github create a new page like this 
https://github.com/Prettyhtml/prettyhtml/issues/new/choose

[the remark this answers is not in the document]
Comment (cemremengu) [reply .github/ISSUE_TEMPLATE.md]: Sorry I am confused (as usual?) 😄 I understand your point but that link is to the `fastify website` project issues so not sure how that will help.

The top part `Please follow one of these issue templates` already links to `/issues/new/choose`

[the remark this answers is not in the document]
Comment (StarpTech) [reply .github/ISSUE_TEMPLATE.md]: OK, I'm fine with that 👍

File: .github/ISSUE_TEMPLATE/regression.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Paste the results here:
+
+```js
+
+```
+
+## Your Environment
+
+- *node version*: 4,5,6,7,8,9
```
Comment (mcollina) [inline .github/ISSUE_TEMPLATE/regression.md]: We support Node 6, 8 and 10.

File: .github/ISSUE_TEMPLATE/regression.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- *fastify version*: >=0.37.0
```
Comment (mcollina) [inline .github/ISSUE_TEMPLATE/regression.md]: Let's start from 1.0.0

File: .github/ISSUE_TEMPLATE/regression.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- *os*: Mac, Windows
```
Comment (mcollina) [inline .github/ISSUE_TEMPLATE/regression.md]: We are missing Linux

File: .github/ISSUE_TEMPLATE/security.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+name: 👮 Security Issue
+about: Responsible Disclosure
+---
+
+## 👮 [Responsible Disclosure](https://github.com/nodejs/security-wg/blob/master/processes/responsible_disclosure_template.md)
+
+Do not open issues that might have security implications. It is critical that security related issues
+are reported privately so we have time to address them before they become public knowledge. Please
+email all security related issues to the [Fastify Team](https://github.com/fastify/fastify#team).
```
Comment (mcollina) [inline .github/ISSUE_TEMPLATE/security.md]: We should a) be recommending to got to hackerone/node security-wg or to email myself or tomas.

Comment (delvedor) [reply .github/ISSUE_TEMPLATE/security.md]: My DM are open on twitter :)
https://twitter.com/delvedor

File: .github/ISSUE_TEMPLATE/question.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+name: 💬 Questions / Help
+about: If you have questions, please check our Gitter or Help repo
+---
+
+## 💬 Questions and Help
+
+### Please note that this issue tracker is not a help form and this issue may be closed.
```
Comment (mcollina) [inline .github/ISSUE_TEMPLATE/question.md]: form -> forum?

File: .github/ISSUE_TEMPLATE/security.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+## 👮 [Responsible Disclosure](https://github.com/nodejs/security-wg/blob/master/processes/responsible_disclosure_template.md)
+
+Do not open issues that might have security implications. It is critical that security related issues
+are reported privately so we have time to address them before they become public knowledge. Please
+notify all security related issues via the following channels:
+
+a) [node-security-wg](https://github.com/nodejs/security-wg)
+
+b) [hackerone](https://www.hackerone.com/)
```
Comment (mcollina) [inline .github/ISSUE_TEMPLATE/security.md]: this is not correct. There is a more detailed process on reporting vulna defined in security-wg. The security-wg uses hackerone.

Comment (cemremengu) [reply .github/ISSUE_TEMPLATE/security.md]: Should I copy the process here or just redirect to security-wg to follow the steps there?

Comment (mcollina) [reply .github/ISSUE_TEMPLATE/security.md]: Just copy https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md#process

**Pull request merged**